torch/distributed/distributed_c10d.py

```diff
@@ -1296,7 +1296,7 @@ def init_process_group(
             )
 
         default_pg, _ = _new_process_group_helper(
-            -1, -1, [], backend, None, group_name, timeout=timeout
+            -1, -1, [], backend, None, group_name, timeout=timeout, group_desc="default_pg"
```

Comment:
nit: would "default" be better? because other descriptor may be like "dp", "tp", "pp", no "_pg" suffix. not a big deal though.